### PR TITLE
Rename components to prepare views for Consensus transactions

### PR DESCRIPTION
**Pull request by buberdds**
Follows pattern that Kristof introduce with Blocks.

### CODE REVIEW

**Comment by csillag** [comment]
These are just renamings, no functional change, right?

File: src/app/pages/SearchResultsPage/SearchResultsList.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { useTranslation } from 'react-i18next'
 import { ResultsGroupByType } from './ResultsGroupByType'
 import { RuntimeBlockDetailView } from '../RuntimeBlockDetailPage'
 import { RouteUtils } from '../../utils/route-utils'
-import { TransactionDetailView } from '../TransactionDetailPage'
+import { RuntimeTransactionDetailPage } from '../RuntimeTransactionDetailPage'
```
Comment (csillag) [inline src/app/pages/SearchResultsPage/SearchResultsList.tsx:6]: I think we are importing the wrong stuff here. (We need the View, but importing the Page)

Comment (buberdds) [reply src/app/pages/SearchResultsPage/SearchResultsList.tsx]: yep I have updated it in the meantime

**Comment by buberdds** [comment]
Only renaming as I don't want to fight with conflicts.

**Review by csillag — APPROVED** [review]
Reviewed, LGTM

**Pull request merged**